src/commands/generate/fromTemplate.ts

```diff
@@ -112,6 +112,16 @@ export default class Template extends Command {
   parser = new Parser();
 
   async run() {
+    try {
+      // Metrics recording when command is invoked
+      await this.recorder.recordActionInvoked('fromTemplate');
```

Comment:
I would aa a prefix to the action including then the `generate` word. Like `generate_from_template` or similar

Reply:
Yeah, makes sense 👍